evalml/pipelines/components/estimators/regressors/arima_regressor.py

```diff
@@ -17,83 +17,89 @@ class ARIMARegressor(Estimator):
     """
     name = "ARIMA Regressor"
     hyperparameter_ranges = {
-        "p": Integer(0, 10),
-        "d": Integer(0, 10),
-        "q": Integer(0, 10),
+        "start_p": Integer(1, 3),
+        "d": Integer(0, 2),
+        "start_q": Integer(1, 3),
+        "max_p": Integer(3, 10),
+        "max_d": Integer(2, 5),
+        "max_q": Integer(3, 10),
+        "seasonal": [True, False]
     }
     model_family = ModelFamily.ARIMA
     supported_problem_types = [ProblemTypes.TIME_SERIES_REGRESSION]
 
-    def __init__(self, date_column=None, trend='n', p=1, d=0, q=0,
-                 random_seed=0, **kwargs):
+    def __init__(self, date_index=None, trend=None, start_p=2, d=0, start_q=2, max_p=5, max_d=2, max_q=5, seasonal=True,
+                 n_jobs=-1, random_seed=0, **kwargs):
         """
         Arguments:
-            date_column (str): Specifies the name of the column in X that provides the datetime objects. Defaults to None.
+            date_index (str): Specifies the name of the column in X that provides the datetime objects. Defaults to None.
             trend (str): Controls the deterministic trend. Options are ['n', 'c', 't', 'ct'] where 'c' is a constant term,
                 't' indicates a linear trend, and 'ct' is both. Can also be an iterable when defining a polynomial, such
                 as [1, 1, 0, 1].
-            p (int or list(int)): Autoregressive order.
-            d (int): Differencing degree.
-            q (int or list(int)): Moving Average order.
+            start_p (int): Minimum Autoregressive order.
+            d (int): Minimum Differencing degree.
```

Comment:
Is there a reason we didn't name this `start_d` to have similar naming conventions as the other variables?

Reply:
The `sktime` implementation of this has decided to name it `d` for some reason so I'm just matching the parameters that will have to be sent to `AutoARIMA`.